conf/application.conf

```diff
@@ -49,6 +49,8 @@ tracking-consent-frontend {
   gtm.container = "a"
 }
 
+emailAuth.generateTestCode = true
```

Comment:
generateTestCode should be false by default. Any application that needs it should override it to true in the environment conf files